GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -112,12 +116,17 @@ - (void)storeEvent:(GDTCOREvent *)event
                  @"The event should have been serialized to bytes");
     NSError *error = nil;
     NSURL *eventFile = [self saveEventBytesToDisk:event eventHash:event.hash error:&error];
-    GDTCORLogDebug("Event saved to disk: %@", eventFile);
-    completion(eventFile != nil, error);
+    if (!eventFile || error) {
+      GDTCORLogDebug("Event failed to save to disk: %@", error);
```

Comment:
It seems like these two log's should be different levels - info versus error

Reply:
Done.

Reply:
I meant - shouldn't this be GDTCORLogError?

Reply:
It is, I think you might be looking at an older set of commits.

![image](https://user-images.githubusercontent.com/1082754/76360637-a04cc600-62da-11ea-8184-ff81e72b89d3.png)